src/QMCWaveFunctions/Fermion/DiracMatrixComputeOMPTarget.hpp

```diff
@@ -220,8 +217,8 @@ class DiracMatrixComputeOMPTarget : public Resource
    *  
    *  \todo measure if using the a_mats without a copy to contiguous vector is better.
    */
-  template<typename TMAT, PlatformKind PL>
-  inline void mw_invertTranspose(compute::Queue<PL>& resource_ignored,
+  template<typename TMAT>
+  inline void mw_invertTranspose(HandleResource& resource,
```

Comment:
This combined with the actual method definition got mw_invert_transpose which turns this into a call where a default constructed HandleResource is stuffed into this call makes me wonder why the resource is in the API here at all.  Looks like with the introduction of the virtual base class this API should drop the resource argument from (mw_invertTranspose)

Reply:
It is not a resource handle but a queue. I should rename the type name and that is it. I will do a few renaming once my series of PRs are settled.

The reason I keep this argument but don't use it is the fact DiracMatrixComputeOMPTarget is a fallback and runs inversion on host and then got transferred. We won't implement a GPU inverter ourselves. Bacause it is a fallback and performance don't matter, we don't care about the queue being passed it. That is how I got intel case working without calling mkl for the batched inversion which will come later.

Reply:
I'm still not sure why this means mw_invertTranspose needs a dead argument.  I can see why mw_invert_transpose does.  Is it going to become live later?

Reply:
I see what you mean now. Removed all the use of dummy queues.